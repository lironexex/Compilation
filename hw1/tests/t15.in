// illegal strings 6 - illegal escape sequence
"this is a legal escape sequence: \t"
"this escape sequence is illegal: \>"
