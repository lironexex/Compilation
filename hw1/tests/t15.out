1 COMMENT 1
2 STRING this is a legal escape sequence: 	
Error undefined escape sequence >
